packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -20,6 +20,7 @@ import 'package:flutter_tools/src/globals.dart' as globals;
 import 'package:flutter_tools/src/reporting/crash_reporting.dart';
 import 'package:flutter_tools/src/reporting/reporting.dart';
 import 'package:flutter_tools/src/runner/flutter_command.dart';
+import 'package:test_api/fake.dart';
```

Comment:
we shouldn't be importing from test_api (except in the few places we *need* to).

```suggestion
import 'package:test/fake.dart';
```